cmake/toolchain.cmake

```diff
@@ -0,0 +1,3 @@
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED OFF)
```

Comment:
Why `OFF`? I guess it should be `ON`.

Reply:
Project owner wanted to use C++11 when  present. Isn't strictly required.